test-requirements.txt

```diff
@@ -1,3 +1,4 @@
+click
```

Comment:
Don't regular requirements get installed in addition to test requirements? If so, I'd think you don't need to put click here since it's part of the usual requirements…